unity-renderer/Assets/DCLPlugins/EmoteAnimations/EmoteAnimationsPlugin.cs

```diff
@@ -7,7 +7,7 @@ public class EmoteAnimationsPlugin : IPlugin
     {
         private readonly EmoteAnimationsTracker emotesAnimationTracker;
 
-        public EmoteAnimationsPlugin() { emotesAnimationTracker = new EmoteAnimationsTracker(DataStore.i.emotes, new EmoteAnimationLoaderFactory(), new WearableItemResolver()); }
+        public EmoteAnimationsPlugin() { emotesAnimationTracker = new EmoteAnimationsTracker(DataStore.i.emotes, new EmoteAnimationLoaderFactory(), new WearableItemResolver(), Environment.i.serviceLocator.Get<IEmotesCatalogService>()); }
```

Comment:
One of the conventions for plugins is that they must be parameterless in construction.

Mockability is not suffering from this decision since plugins are barely entry points which near to none testing code. In this case, for example, everything is done by the Tracker which is indeed using dependency injection.

Reply:
ok!